src/engraving/libmscore/textbase.cpp

```diff
@@ -252,7 +252,9 @@ RectF TextCursor::cursorRect() const
     mu::draw::Font _font  = fragment ? fragment->font(_text) : _text->font();
     if (_font.family() == _text->score()->styleSt(Sid::MusicalSymbolFont)) {
         _font.setFamily(_text->score()->styleSt(Sid::MusicalTextFont), draw::Font::Type::MusicSymbolText);
-        _font.setPointSizeF(fragment->format.fontSize());
+        if (fragment) {
```

Comment:
As far as I know, it's a legitimate situation that `fragment` is null, namely when the text is empty because it's just created. So this fix is not simply hiding a deeper problem :)